test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -171,8 +171,10 @@ static odp_crypto_session_t session_create(odp_crypto_op_t op,
 
 	/* Create a crypto session */
 	odp_crypto_session_param_init(&ses_params);
-	ses_params.op = op;
-	ses_params.op_type = op_type;
+	ses_params.op = session->op;
+	ses_params.op_type = session->op_type;
+	ses_params.cipher_range_in_bits = session->cipher_range_in_bits;
+	ses_params.auth_range_in_bits = session->auth_range_in_bits;
 	ses_params.auth_cipher_text = (order == AUTH_CIPHERTEXT);
```

Comment:
Does it make sense to move order also to crypto_session_t?

Reply:
Perhaps, and the same for hash_mode, but since the change would not really be related to any of the commits in this PR it does not need to be in this PR.

Reply:
Agreed.